### DISCUSSION TITLE
Alert / Info / Warning / Error div

### DISCUSSION THREAD

**Discussion by madcoda9000** [microsoft/fluentui-blazor#641] [Q&A] [answered]
Hello,

as the css color variables existing already, i am wondering on how to make an alert div like erro, warning etc...

Are the any pre defined css classes for such cases? like it is done in bootstrap for example?

 Best Ragrds
madcoda9000

**Comment by vnbaaij** [comment, marked as answer]
Yes, those variables already exist. See https://github.com/microsoft/fluentui-blazor/blob/new-components/src/Core/wwwroot/css/variables.css 

We are also going to add a `<FluentAlert>` component (similar to the FluentToast), Not sure yet if it will be in v3 or that it will be added in v3.1
